api/controllers/UserController.js

```diff
@@ -2559,4 +2559,123 @@ module.exports = {
     );
     return user;
   },
+
+  /*
+   * @api [delete] /user/{id}/clients/{client}
+   * tags:
+   *   - user
+   * summary: Revokes one OAuth Client from a user's profile.
+   * responses:
```

Comment:
Wouldn't be better to add the `id` and `client` parameters here to match lines  2585 and 2586? That would also help when trying the endpoint in swagger.

Reply:
Great idea! I never quite understood that we can document params in the path, but it was [easy to find out how](https://swagger.io/docs/specification/describing-parameters/) and allows the API call to be tested in our Swagger site.